azure/template.json

```diff
@@ -57,7 +57,7 @@
     },
     "functionsExtensionVersion": {
       "type": "string",
-      "defaultValue": "~2.0"
+      "defaultValue": "~3.0"
```

Comment:
This just needs updating to "~3" please.

Reply:
changed.

Reply:
Thanks @shomavg